packages/nocodb/src/lib/Noco.ts

```diff
@@ -496,34 +497,34 @@ export default class Noco {
 
   private async initJwt(): Promise<any> {
     if (this.config?.auth?.jwt) {
-      if (!this.config.auth.jwt.secret) {
-        let secret = (
-          await Noco._ncMeta.metaGet('', '', 'nc_store', {
-            key: 'nc_auth_jwt_secret',
-          })
-        )?.value;
-        if (!secret) {
-          await Noco._ncMeta.metaInsert('', '', 'nc_store', {
-            key: 'nc_auth_jwt_secret',
-            value: (secret = uuidv4()),
-          });
-        }
-        this.config.auth.jwt.secret = secret;
+      let secret = (
+        await Noco._ncMeta.metaGet('', '', MetaTable.STORE, {
+          key: 'nc_auth_jwt_secret',
+        })
+      )?.value;
+
+      if (!secret) {
+        await Noco._ncMeta.metaInsert('', '', MetaTable.STORE, {
+          key: 'nc_auth_jwt_secret',
+          value: this.config.auth.jwt.secret ?? (secret = uuidv4()),
+        });
```

Comment:
I think storing the secret key from env is not a good idea. If someone got the DB access they could easily decrypt the project config and password, etc. By default, we are generating and storing the key if env is missing but if env is defined we should use it and ignore saving it in my opinion. @o1lab  - any thoughts